lm_eval/models/vllm_causallms.py

```diff
@@ -176,23 +179,46 @@ def max_length(self):
     def max_gen_toks(self):
         return self._max_gen_toks
 
+    def apply_chat_template(self, chat_history: List[Dict[str, str]]) -> str:
+        """
+        Method to apply a chat template to a list of chat history between user and model.
+        """
+        return self.tokenizer.apply_chat_template(
+            chat_history, tokenize=False, add_generation_prompt=True
+        )
+
+    @property
+    def chat_template(self) -> str:
+        if self.tokenizer.chat_template is not None:
+            return self.tokenizer.chat_template
+        return self.tokenizer.default_chat_template
```

Comment:
This `default_chat_template` behavior should carry over to HFLM as well I believe!

Reply:
This is from `HFLM`!

Reply:
oops, swear I looked and it wasn't there in HFLM, my bad